options/locale/locale_en-US.ini

```
@@ -370,6 +370,9 @@ show_only_public = Showing only public
 
 issues.in_your_repos = In your repositories
 
+no_repos = No repositories
+no_orgs = No organization
```

```suggestion
no_orgs = No organizations
```

Plural on both, right?